source/atlas-cli-deploy-local.txt

```diff
@@ -86,6 +86,105 @@ Before you begin, complete the following prerequisites:
 
 .. include:: /includes/steps-atlas-cli-deploy-local-prereqs.rst
 
+.. _atlas-cli-pvt-registry-setup:
+
+(Optional) Set up a Private Image Registry 
+------------------------------------------
+
+This section demonstrates how to pull and tag images, load images to a
+container registry, and configure the {+atlas-cli+} to use private
+images for building local clusters. 
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the Docker command line tools.
+   
+   .. step:: Create your container registry with the namespace and name of repository. 
```

Comment:
Minor: I wonder if we just need to add this as a prerequisite upfront rather than an explicit step? Something like "Prerequisite: a container registry for which you have write permission". No strong feeling though.